ow_sim_tests/scripts/test_arm_stop.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import roslib
+import unittest
+import owl_msgs.msg
+import actionlib
+
+from action_testing import *
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'arm_stop'
+roslib.load_manifest(PKG)
+
```

Comment:
```suggestion

# NOTE: Running this test at 3x real-time factor results in a slightly different
# log outcome, but otherwise the test still succeeds. This should be fixed by
# OW-1193

```